src/main/java/org/sefglobal/scholarx/util/EmailUtil.java

```diff
@@ -1,42 +1,27 @@
 package org.sefglobal.scholarx.util;
 
-import org.sefglobal.scholarx.model.Email;
-import org.springframework.mail.javamail.JavaMailSender;
-import org.springframework.mail.javamail.MimeMessageHelper;
-import org.springframework.stereotype.Component;
-import org.thymeleaf.context.Context;
-import org.thymeleaf.spring5.SpringTemplateEngine;
+import org.sefglobal.scholarx.model.Mail;
+import org.sefglobal.scholarx.service.MailConnection;
+import org.simplejavamail.api.email.Email;
+import org.simplejavamail.api.mailer.Mailer;
+import org.simplejavamail.email.EmailBuilder;
 
-import javax.mail.MessagingException;
-import javax.mail.internet.MimeMessage;
-import java.nio.charset.StandardCharsets;
-
-
-@Component
 public class EmailUtil {
+    private final MailConnection mailConnection;
 
-    private final JavaMailSender emailSender;
-
-    private final SpringTemplateEngine templateEngine;
-
-    public EmailUtil(JavaMailSender emailSender, SpringTemplateEngine templateEngine) {
-        this.emailSender = emailSender;
-        this.templateEngine = templateEngine;
+    public EmailUtil(MailConnection mailConnection) {
+        this.mailConnection = mailConnection;
     }
 
-    public void sendSimpleMessage(Email email) throws MessagingException {
-        MimeMessage message = emailSender.createMimeMessage();
-        MimeMessageHelper helper = new MimeMessageHelper(message, MimeMessageHelper.MULTIPART_MODE_MIXED_RELATED, StandardCharsets.UTF_8.name());
-
-        Context context = new Context();
-        context.setVariables(email.getProps());
-
-        String html = templateEngine.process("scholarx", context);
+    public void sendEmail(Mail mail) {
 
-        helper.setTo(email.getEmail());
-        helper.setSubject(email.getSubject());
-        helper.setText(html, true);
+        Email email = EmailBuilder.startingBlank()
+                .to(mail.getEmailAddress())
+                .withSubject(mail.getSubject())
+                .withHTMLText(mail.getMessage())
+                .buildEmail();
 
-        emailSender.send(message);
+        Mailer mailer = mailConnection.getInstance();
```

Comment:
`.getInstance` is a static method. You don't need to create an instance to access a static method.